src/renderers/native/__tests__/ReactNativeAttributePayload-test.js

```diff
@@ -15,227 +15,240 @@ var ReactNativePropRegistry = require('ReactNativePropRegistry');
 var diff = ReactNativeAttributePayload.diff;
 
 describe('ReactNativeAttributePayload', () => {
-
   it('should work with simple example', () => {
-    expect(diff(
-      {a: 1, c: 3},
-      {b: 2, c: 3},
-      {a: true, b: true}
-    )).toEqual({a: null, b: 2});
+    expect(diff({a: 1, c: 3}, {b: 2, c: 3}, {a: true, b: true})).toEqual({
+      a: null,
+      b: 2,
+    });
   });
 
   it('should skip fields that are equal', () => {
-    expect(diff(
-      {a: 1, b: 'two', c: true, d: false, e: undefined, f: 0},
-      {a: 1, b: 'two', c: true, d: false, e: undefined, f: 0},
-      {a: true, b: true, c: true, d: true, e: true, f: true}
-    )).toEqual(null);
+    expect(
+      diff(
+        {a: 1, b: 'two', c: true, d: false, e: undefined, f: 0},
+        {a: 1, b: 'two', c: true, d: false, e: undefined, f: 0},
+        {a: true, b: true, c: true, d: true, e: true, f: true},
+      ),
+    ).toEqual(null);
   });
 
   it('should remove fields', () => {
-    expect(diff(
-      {a: 1},
-      {},
-      {a: true}
-    )).toEqual({a: null});
+    expect(diff({a: 1}, {}, {a: true})).toEqual({a: null});
   });
 
   it('should remove fields that are set to undefined', () => {
-    expect(diff(
-      {a: 1},
-      {a: undefined},
-      {a: true}
-    )).toEqual({a: null});
+    expect(diff({a: 1}, {a: undefined}, {a: true})).toEqual({a: null});
   });
 
   it('should ignore invalid fields', () => {
-    expect(diff(
-      {a: 1},
-      {b: 2},
-      {}
-    )).toEqual(null);
+    expect(diff({a: 1}, {b: 2}, {})).toEqual(null);
   });
 
   it('should use the diff attribute', () => {
     var diffA = jest.fn((a, b) => true);
     var diffB = jest.fn((a, b) => false);
-    expect(diff(
-      {a: [1], b: [3]},
-      {a: [2], b: [4]},
-      {a: {diff: diffA}, b: {diff: diffB}}
-    )).toEqual({a: [2]});
+    expect(
+      diff(
+        {a: [1], b: [3]},
+        {a: [2], b: [4]},
+        {a: {diff: diffA}, b: {diff: diffB}},
+      ),
+    ).toEqual({
+      a: [2],
+    });
```

Comment:
Oh, I think I know what happened. You started with 120 columns which formatted it as

```js
expect(diff(...)).toEqual({
  a: [2],
});
```

and we have a rule that says if an object was expanded, we keep it expanded, so when you moved it back to 80 columns, the diff got expanded and this one stayed expanded as well.

That rule is not ideal but this is the best trade-off we could find as there are valid reasons why objects could be inlined or expanded depending on the context.